cxx4opencl/address_spaces.txt

```diff
@@ -185,19 +260,24 @@ __kernel void bar() {
   c1.foo(); // will resolve to the first 'foo'.
   c2.foo(); // will resolve to the second 'foo'.
   c3.foo(); // error due to mismatching address spaces - can't convert to
-            // '__local' or generic addr spaces.
+            // '__local' or generic/'__private' addr spaces.
```

Comment:
It may be the only place where you use the contracted form "addr".
```suggestion
            // '__local' or generic/'__private' address spaces.
```